navigation.html

```diff
@@ -0,0 +1,148 @@
+         </nav>
+   </div>
+   <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%2851%2C51%2C51%2C1%29&t=blackboard&wt=none&l=text%2Fx-c%2B%2Bsrc&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%2520%2520%2520%253Cdiv%2520class%253D%2522nav-component2%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cnav%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522nav1-list%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cul%2520class%253D%2522topnav-sample%2522%253E%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253E%253Cimg%2520src%253D%2522https%253A%252F%252Fwww.freepnglogos.com%252Fuploads%252Fshopify-logo-png%252Fshopify-bag-logo-png-0.png%2522%253E%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253E%253Ca%2520class%253D%2522active%2522%2520href%253D%2522%2523home%2522%2520class%253D%2522left%2522%253EHome%253C%252Fa%253E%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253E%253Ca%2520href%253D%2522%2523products%2522%2520class%253D%2522left%2522%253EProducts%253C%252Fa%253E%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253E%253Ca%2520href%253D%2522%2523contact%2522%2520class%253D%2522left%2522%253EContact%253C%252Fa%253E%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Csvg%253E%2520%252F%252FuserIcon%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fsvg%253E%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253E%253Cbutton%2520class%253D%2522sign-out-btn%2522%253ESign%2520out%253C%252Fbutton%253E%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Ful%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fnav%253E%250A%2520%2520%2520%253C%252Fdiv%253E"
+  style="width: 885px; height: 582px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin" class="img-nav">
+</iframe>
+         
+<p class="navtext-text">Contains the sub-nav categories under main nav tabs. Useful for bigger Applications and empowers a good UX.</p>
+    <div class="nav-component">
+        
+
+<div class="navbar-sample">
+    <a href="index.html">Home</a>
+    <div class="subnav-sample">
+      <button class="subnavbtn">About <i class="fa fa-caret-down"></i></button>
+      <div class="subnav-content">
+        <a href="#company">Company</a>
+        <a href="#team">Team</a>
+        <a href="#careers">Careers</a>
+      </div>
+    </div> 
+    <div class="subnav-sample">
+      <button class="subnavbtn">Services <i class="fa fa-caret-down"></i></button>
```

Comment:
on hover, this button background color is not covering the entire width of the button.